bolt.gemspec

```diff
@@ -53,7 +53,7 @@ Gem::Specification.new do |spec|
 
   # there is a bug in puppetlabs_spec_helper for modules without fixtures
   spec.add_development_dependency "bundler", ">= 1.14"
-  spec.add_development_dependency "puppetlabs_spec_helper", "~> 2.6"
+  spec.add_development_dependency "puppetlabs_spec_helper", "~> 2.7"
   spec.add_development_dependency "rake", "~> 10.0"
```

Comment:
Upgrading rake version is included in https://github.com/puppetlabs/bolt/pull/1030/commits/dd5d7c6512bcfde608eb17fe339f17a65b5482bf#diff-83c8d575bd9e6c82d40395cc2d9f2115R57